pages/common/rga.md

```diff
@@ -0,0 +1,24 @@
+# rga
+
+> Ripgrep wrapper with rich file type searching capabilities.
+> More information: <https://github.com/phiresky/ripgrep-all>.
+
+- Recursively search the current directory for a regular expression:
+
+`rga {{regular_expression}}`
+
+- List all known adapters:
+
+`rga --rga-list-adapters`
+
+- Change which adapters to use (e.g. ffmpeg, pandoc, poppler etc.):
+
+`rga --rga-adapters={{adapter1,adapter2}} {{regular_expression}}`
+
+- Use more accurate but slower matching by mime type:
+
+`rga --rga-accurate {{regular_expression}}`
+
+- Show detailed help:
```

Comment:
We use this usually
```suggestion
- Display help:
```

Reply:
There are two different `help` options on `rga`. According to the `-h`elp:
```
-h shows a concise overview
--help shows more detail and advanced options
```

What about:
```suggestion
- Display detailed help:
```

Reply:
Or maybe
```suggestion
- Display long help:
```